libs/ha/ha_common_libs_k8s.py

```diff
@@ -1824,46 +1825,52 @@ def iam_bucket_cruds(self, event, s3_obj, user_crud=False, num_users=None, bkt_c
         :param num_users: Number of iam users to be created and deleted
         :param bkt_crud: Flag for performing bucket crud operations
         :param num_bkts: Number of buckets to be created and deleted
+        :param del_users_dict: Dict of users to be deleted
         :param output: Output queue in which results should be put
         :return: Queue containing output lists
         """
         exp_fail = list()
         failed = list()
+        created_users = list()
         user_del_failed = list()
         user = None
+        del_users = list(del_users_dict.keys()) if del_users_dict else list()
         if user_crud:
             LOGGER.info("Create and delete %s IAM users in loop", num_users)
-            for i in range(num_users):
+            for i_i in range(num_users):
                 try:
-                    LOGGER.debug("Creating %s user", i)
+                    LOGGER.debug("Creating %s user", i_i)
                     user = None
                     user = self.mgnt_ops.create_account_users(nusers=1)
                     if user is None:
                         if event.is_set():
                             exp_fail.append(user)
                         else:
                             failed.append(user)
-                        break
-                    LOGGER.debug("Deleting %s user", i)
-                    resp = self.delete_s3_acc_buckets_objects(user)
-                    if not resp[0]:
-                        user_del_failed.append(user)
-                        if event.is_set():
-                            exp_fail.append(user)
-                        else:
-                            failed.append(user)
                     else:
-                        LOGGER.debug("Created and deleted %s user successfully", i)
+                        created_users.append(user)
+                        LOGGER.debug("Created %s user successfully", i_i)
```

Comment:
same as here..

or we can just print the list of user created in the end of the loop with count

Reply:
This will help while debugging issue, I will keep one of the log lines.